src/screens/Messages/List/ChatListItem.tsx

```diff
@@ -29,6 +29,13 @@ export function ChatListItem({
   const {currentAccount} = useSession()
   const menuControl = useMenuControl()
   const {gtMobile} = useBreakpoints()
+  const otherUser = convo.members.find(
+    member => member.did !== currentAccount?.did,
+  )
+  const isDeletedAccount = otherUser?.handle === 'missing.invalid'
+  const displayName = isDeletedAccount
+    ? 'Deleted Account'
+    : otherUser?.displayName ?? otherUser?.handle
```

Comment:
Should be
```js
  const displayName = isDeletedAccount
    ? 'Deleted Account'
    : otherUser?.displayName || otherUser?.handle
```

Reply:
How do you get it to do the "accept suggested changes" button?

Reply:
Why bluesky why